src/overlays/actors/ovl_En_Osn/z_en_osn.c

```diff
@@ -74,59 +101,841 @@ static DamageTable D_80AD2550 = {
+                if ((gSaveContext.day == 3) && (gSaveContext.time >= CLOCK_TIME(5, 0)) &&
+                    (gSaveContext.time < CLOCK_TIME(6, 0))) {
+                    return 0x2006;
+                }
+                return 0x1FCD;
+            }
+            this->unk_1EA |= 4;
+            return 0x1FC8;
+        }
+
+        if (player->transformation == PLAYER_FORM_GORON) {
+            if (this->unk_1EA & 8) {
+                this->unk_1EA |= 0x20;
+                if ((gSaveContext.day == 3) && (gSaveContext.time >= CLOCK_TIME(5, 0)) &&
+                    (gSaveContext.time < CLOCK_TIME(6, 0))) {
+                    return 0x2006;
+                }
+                return 0x1FCD;
+            }
+            this->unk_1EA |= 8;
+            if (gSaveContext.weekEventReg[0x4C] & 0x20) {
+                return 0x1FC8;
+            }
+
+            gSaveContext.weekEventReg[0x4C] |= 0x20;
+            return 0x1FCE;
+        }
+
+        if (player->transformation == PLAYER_FORM_ZORA) {
+            if (this->unk_1EA & 0x10) {
+                this->unk_1EA |= 0x20;
+                if ((gSaveContext.day == 3) && (gSaveContext.time >= CLOCK_TIME(5, 0)) &&
+                    (gSaveContext.time < CLOCK_TIME(6, 0))) {
+                    return 0x2006;
+                }
+                return 0x1FCD;
+            }
+            this->unk_1EA |= 0x10;
+            if (gSaveContext.weekEventReg[0x4C] & 0x40) {
+                return 0x1FC8;
+            }
+            gSaveContext.weekEventReg[0x4C] |= 0x40;
+            return 0x1FD0;
+        }
+
+        if (Player_GetMask(globalCtx) == PLAYER_MASK_NONE) {
+            if (this->unk_1EA & 2) {
+                this->unk_1EA |= 0x20;
+                if ((gSaveContext.day == 3) && (gSaveContext.time >= CLOCK_TIME(5, 0)) &&
+                    (gSaveContext.time < CLOCK_TIME(6, 0))) {
+                    return 0x2006;
+                }
+                return 0x1FCD;
+            }
+            this->unk_1EA |= 2;
+            return 0x1FC8;
+        }
+
+        return func_80AD0B38(this, globalCtx);
+    }
+
+    this->unk_1EA |= 0x20;
+    if ((gSaveContext.day == 3) && (gSaveContext.time >= CLOCK_TIME(5, 0)) && (gSaveContext.time < CLOCK_TIME(6, 0))) {
+        return 0x2004;
+    }
+
+    return 0x1FAE;
+}
+
+void func_80AD10FC(EnOsn* this, GlobalContext* globalCtx) {
+    switch (this->unk_1F4) {
+        case 0x1FC8:
+            this->unk_1F4 = 0x1FC9;
+            break;
+
+        case 0x1FC9:
+            this->unk_1F4 = 0x1FCA;
+            break;
+
+        case 0x1FCA:
+            if ((gSaveContext.day == 3 && gSaveContext.time >= CLOCK_TIME(5, 0)) &&
+                gSaveContext.time < CLOCK_TIME(6, 0)) {
+                this->unk_1F4 = 0x2007;
+            } else {
+                this->unk_1F4 = 0x1FCB;
+            }
+            break;
+
+        case 0x1FCB:
+
+        case 0x2007:
+            this->unk_1F4 = 0x1FCC;
```

Comment:
```suggestion
        case 0x1FCB:
        case 0x2007:
            this->unk_1F4 = 0x1FCC;
```
When you have two cases that fall through like this, you don't need a space between them. There's a few in this switch function that this comment applies to